ui/pages/send/send-content/send-to/send-to.component.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import { connect, useSelector, useDispatch } from 'react-redux';
+import { withRouter } from 'react-router-dom';
+import { compose } from 'redux';
+import { useI18nContext } from '../../../../hooks/useI18nContext';
+import CloseIcon from '../../../../components/ui/icon/close-icon.component';
+import { getRecipient, resetRecipientInput } from '../../../../ducks/send';
+import { ellipsify } from '../../send.utils';
+
+function mapStateToProps(state) {
+  return {};
+}
+
+function SendTo({}) {
+  const t = useI18nContext();
+  const dispatch = useDispatch();
+  const recipient = useSelector(getRecipient);
+  return (
+    <div className={classnames('send-v2__to-wrap')}>
+      <div className="send-v2__to-title">To</div>
+      <div className="send-v2__to-cont">
+        <div className="send-v2__to-cont-img">pic</div>
```

Comment:
- Should be `Identicon` for now

Reply:
fixed in bd5a9d52dfae71fc07e4aa91d9c7d3d7ede5c444

Reply:
- Verified